agent/tests/test_trade.py

```
@@ -185,18 +190,21 @@ def test_sell_all_zero_balance(self, MockClient, MockAgent, MockCanister,
 
 class TestRunTradeErrors:
     def test_no_wallet(self, odin_project_no_wallet):
+        """Verify no wallet."""
         from iconfucius.cli.trade import run_trade
         result = run_trade(bot_name="bot-1", action="buy", token_id="29m8", amount="1000")
         assert result["status"] == "error"
         assert "wallet" in result["error"].lower()
 
     def test_invalid_action(self, odin_project):
```

Ruff (0.15.2)

[warning] Unused method argument: `odin_project`

(ARG002)